Lib/Shim.php

```diff
@@ -22,6 +22,13 @@ class Shim {
 	 */
 	const VALIDATE_PROPERTY = 'Shim.warnAboutValidateProperty';
 
+	/**
+	 * The name of the shim that checks the usage of the deprecated Model::bindModel() method.
+	 *
+	 * @var string
```

Comment:
those var are not necessary for consts.

Reply:
Okay. Eclipse generated them automatically.

Reply:
Usually `@var` is used in the core - https://github.com/cakephp/cakephp/search?p=1&q=const&type=&utf8=%E2%9C%93.

Reply:
"You may use the"
And for const it doesnt really help, as it is clear from the definition what type it is, it cannot change anymore at runtime.

Reply:
Of course, it is optional. Although as IDEs add them automatically I would not fight against that.

Reply:
Not if properly configured.